R/quosure.R

```diff
@@ -211,6 +217,13 @@ quo_label <- function(quo) {
 quo_text <- function(quo, width = 60L, nlines = Inf) {
   expr_text(quo_expr(quo), width = width, nlines = nlines)
 }
+#' @export
+quo_name <- function(quo) {
+  if (!is_symbol(quo)) {
+    abort("`quo` must quote a symbol")
```

Comment:
It's used when you want to turn an expression into a name that you can use for (e.g.) a dplyr column (or can easily be pasted together with something else). I think `expr_label()` is fine as is : it's a label for use in error messages.